src/reference/informative/informC08_interpretation_of_mathematics3.rst

```diff
@@ -0,0 +1,164 @@
+.. _informC08_interpretation_of_mathematics3:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding how the units work
+
+    We know that units in real life make a big difference, but in the CellML specification universe, they don't. 
```

Comment:
They do for connections

Reply:
I think I say that somewhere, will make it more clear her too, thanks.

Reply:
16fef23